docs/installation.md

```diff
@@ -50,91 +50,86 @@ docker-compose up --detach
+    --restart <OPTIONAL: restart the partitioning process from the beginning, otherwise restarts from the last created chunk. Default = False> # If true, the partitioning process restarts instead using a checkpoint.
+    --workers <OPTIONAL: number of concurrent workers to use. Default = 256> # The number of concurrent workers to use to generate the partition.
+    --subtensor.network <OPTIONAL: the bittensor chain endpoint. Default = finney> # The chain endpoint to use to generate the partition.
+    --wallet.name <OPTIONAL: your miner wallet. Default = default> # Must be created using the bittensor-cli, btcli w new_coldkey.
+    --wallet.hotkey <OPTIONAL: your validator hotkey. Defautl = default> # Must be created using the bittensor-cli, btcli w new_hotkey.
 ```
 
-## Register your hotkey
-You can find steps here on [Official Bittensor Documentation](https://docs.bittensor.com/subnets/register-and-participate])
-
-## Mining
-
-To run the miner
+Example:
 ```bash
-pm2 start neurons/miner.py --name miner --interpreter python3 -- 
-    --wallet.name <OPTIONAL: your miner wallet, default = default> # Must be created using the bittensor-cli, btcli wallet new_coldkey
-    --wallet.hotkey <OPTIONAL: your validator hotkey, defautl = default> # Must be created using the bittensor-cli btcli wallet new_hotkey
-    --db_root_path <OPTIONAL: path where you want the DB files stored, default = "~/bittensor-db">  # This is where the partition will be created storing network data.
-    --logging.debug # Run in debug mode, alternatively --logging.trace for trace mode
-    --threshold <OPTIONAL: threshold i.e. 0.9, default =  0.9>  # The threshold for the partitioning algorithm which is the maximum amount of space the miner can use based on available.
-    --netuid <OPTIONAL: the subnet netuid, defualt = 7> # This is the netuid of the storage subnet.
-    --subtensor.network local # <OPTIONAL: the bittensor chain endpoint, default = finney, local, test> : The chain endpoint to use to generate the partition.  (highly recommend running subtensor locally)
-    --steps_per_reallocate <OPTIONAL: the number of steps before reallocating, default = 1000> # The number of steps before reallocating.
-    # --restart <OPTIONAL: restart the partitioning process from the beginning, otherwise restarts from the last created chunk. default = False> # If true, the partitioning process restarts instead using a checkpoint.
+python3 neurons/allocate.py
+    --db_root_path ~/subnet07-db
+    --size_in_gb 5000
+    --disable_verify
+    --workers 8
+    --subtensor.network local
+    --wallet.name coldkey
+    --wallet.hotkey hotkey1
 ```
 
-- Example 1 (with default values):
+## Register your hotkey
+You can find steps here on [Official Bittensor Documentation](https://docs.bittensor.com/subnets/register-and-participate]).
+
+## Validating
+To run the validator.
```

Comment:
Could you explain it better? I don't understand.

Reply:
Validators need to specify `--validator` flag so that they generate hash only.

Reply:
Yes, we had the parameter before, but I deleted it after read your comment:
> They're usually miners who run allocation before running miner, so better to remove this flag.

It's true that it can be useful, but in reality only miners will use the `allocate.py` module directly. When the module is invoked from `validator.py`, the `only_hash` parameter of the `generate` and `run_rust_generate` methods is set to `True`, so there is no problem generating the database for the validators .